nixos/tests/omnom.nix

```diff
@@ -26,17 +31,116 @@ in
             server.address = "0.0.0.0:${toString servicePort}";
           };
         };
+
+        environment.systemPackages =
+          let
+            # https://addons.mozilla.org/en-US/firefox/addon/omnom/
+            addon = pkgs.fetchurl {
```

Comment:
The derivation already [includes the addon](https://github.com/NixOS/nixpkgs/blob/30e2e2857ba47844aa71991daa6ed1fc678bcbb7/pkgs/by-name/om/omnom/package.nix#L75). Can't we just use that?

Reply:
I think I've missed that, will look into it.

Reply:
The extension in the package is broken, I'll fix that first.

Reply:
https://github.com/NixOS/nixpkgs/pull/421398